templates/heat/config/heat.conf

```diff
@@ -49,3 +49,7 @@ user_domain_name=Default
 project_name=service
 project_domain_name=Default
 interface=internal
+
+[clients]
+endpoint_type=internal
+insecure=true
```

Comment:
the insecure should be not needed, right? when we enable tls for internal endpoints we also add the CAs to the deployment pods

Reply:
Yeah we can drop it as `insecure=false` is the default. However, insecure=false only makes sense when there are certs to be validated.